thunder/core/jit_ext.py

```diff
@@ -1521,14 +1521,29 @@ def process_recorded_modifications(ctx, epilogue_trace):
 
 
 def bind_inputs(name, trace, input_vars, input_proxies):
+    from thunder.core import utils
+
     # Unpacks inputs into the computation trace
     # TODO This currently does the unpacks at the end of the trace, then moves them to the beginning, there's
     #   almost certainly a more elegant way to do this
-    with tracectx(trace):
-        p: Proxy
-        for p in input_proxies:
-            prims.unpack_trivial(p, name=p.name)
-
+    num_orig_bsyms = len(trace.bound_symbols)
+
+    # note(crcrpar): The path looks neat but it does not work for a trace
```

Comment:
Where is this called? Is it still needed now that functionalization is moved into `thunder.jit` to be applied after the interpreter?

Reply:
no, great catch. reverted the changes in this file